src/zope/testrunner/tests/testrunner-subunit-v2.rst

```diff
@@ -291,8 +291,8 @@ https://bugs.launchpad.net/subunit/+bug/1740158.)
     True
 
 
-Layers that can't be torn down
-------------------------------
+Layer failures
+--------------
```

Comment:
(Reviewing code while tired is not a good idea: I've feelings, but no articulatory power to express them well.)

To me it's not a *failure* when a layer's tearDown raises NotImplementedError -- it's an explicitly supported way of stating "tear down is not supported by this layer."  (It's a rather strange way -- I'd've preferred the layer to set tearDown=None, or something, so this fact could be discovered in advance and the test runner could punish the bad layer by running it in a subprocess, instead of running all the other innocent layers, but I digress.)

What I'm saying, I'd prefer the section about layer failures to be separate from layers that don't support teardown.

Reply:
OK, I've moved this into its own section.